From fd5c3034fcd98cc741f2768871074060d2c08ef1 Mon Sep 17 00:00:00 2001
From: Victor Gomes <victorgomes@chromium.org>
Date: Wed, 24 Jun 2026 09:04:37 +0200
Subject: [PATCH] [wasm] Add missing include for std::atomic

Upstream patch from: http://cl/933854604

Bug: 524928531
Change-Id: I6fa6d5beaae9cc7dd51c8200b12d1acb90da0b56
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/7991975
Reviewed-by: Daniel Lehmann <dlehmann@chromium.org>
Commit-Queue: Daniel Lehmann <dlehmann@chromium.org>
Auto-Submit: Victor Gomes <victorgomes@chromium.org>
Commit-Queue: Victor Gomes <victorgomes@chromium.org>
Cr-Commit-Position: refs/heads/main@{#108199}
---
 src/wasm/pgo.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/wasm/pgo.h b/src/wasm/pgo.h
index bfc19f58678f..28e6b6f637c0 100644
--- a/src/wasm/pgo.h
+++ b/src/wasm/pgo.h
@@ -9,6 +9,7 @@
 #error This header should only be included if WebAssembly is enabled.
 #endif  // !V8_ENABLE_WEBASSEMBLY
 
+#include <atomic>
 #include <vector>
 
 #include "src/base/vector.h"
